scvi/models/log_likelihood.py

```diff
@@ -96,7 +105,10 @@ def log_nb_positive(x, mu, theta, eps=1e-8):
     if theta.ndimension() == 1:
         theta = theta.view(1, theta.size(0))  # In this case, we reshape theta for broadcasting
 
-    res = theta * torch.log(theta + eps) - theta * torch.log(theta + mu + eps) + x * torch.log(
-        mu + eps) - x * torch.log(theta + mu + eps) + torch.lgamma(x + theta) - torch.lgamma(
-        theta) - torch.lgamma(x + 1)
+    res = theta * (torch.log(theta + eps) - torch.log(theta + mu + eps)) + \
+        x * (torch.log(mu + eps) - torch.log(theta + mu + eps)) + \
+        torch.lgamma(x + theta) - \
+        torch.lgamma(theta) - \
+        torch.lgamma(x + 1)
```

Comment:
Should we add some named variable here too ?

Reply:
Yeah good catch, I've added `log_theta_mu_eps = torch.log(theta + mu + eps)` now